### ISSUE TITLE
margin at 1350

### ISSUE THREAD

**Issue by jeffalo** [jeffalo/ocular#57]

**Comment by jeffalo** [comment]
i was referring to the margin breakpoints. ocular is a bit wide on smaller screened devices.

**Comment by CluckCluckChicken** [comment]
interesting

**Comment by towerofnix** [comment]
i browse basically every internet page at 120-150% resolution because my eyes suck and dear lord the margin makes it kinda hard to read forum posts lol, they take up the full width of my desktop screen:

![A forum thread I've opened in Ocular, with the width so wide you have to scan your eyes across nearly the entire screen reading each text line.](https://user-images.githubusercontent.com/9948030/120115244-2f9cfe00-c159-11eb-893c-c5322f795d78.png)

**Comment by jeffalo** [comment]
so @towerofnix, should the margin breakpoint be smaller? how wide is your monitor?

**Comment by towerofnix** [comment]
@jeffalo my monitor is `1920px` wide, though at full-screen the CSS width is about `1272px`. the trouble is that this is an *apparently* thin amount of space, but the width of the screen its displaying on is still, like, my entire 20 inch screen or so :P

something ive seen a few layouts do is go for a percentage-based look instead of static margins, allowing the breakpoint to be placed quite a bit lower before the margin becomes an obstruction; e.g. see `margin-left: 12%; margin-right: 12%` at [100%](https://user-images.githubusercontent.com/9948030/120313888-fd9eaf80-c2b0-11eb-9868-a0f4331aed67.png), [150%](https://user-images.githubusercontent.com/9948030/120313919-08594480-c2b1-11eb-814f-63085c3dad68.png), [100% (small screen)](https://user-images.githubusercontent.com/9948030/120313957-11e2ac80-c2b1-11eb-8a98-0f08ea92d3b9.png), and [150% (small screen)](https://user-images.githubusercontent.com/9948030/120313965-13ac7000-c2b1-11eb-9568-d59b0ed094bb.png).

it only really becomes a bother at the final smallest screen (150% zoom thin viewport); that could possibly be detected (and have padding removed) with `in`/`cm` units, though i havent played with it myself :P

worth noting that scratch uses roughly this layout:

```
max-width: 1350px; /* per memory, check scratch's css to be sure */
margin-left: auto;
margin-right: auto;
```

but it also has a higher default font size than ocular, so that counters out the relatively large width of 1350. just various factors to consider when layouting!

**Comment by jeffalo** [comment]
so if i mirroed the scratch forum's margins would that be good enough?

```css
.margined {
  width: 942px;
  margin-left: auto;
  margin-right: auto;
}
```

and the scratch forum doesn't do this, but it fixes margins on mobile

```css
@media only screen and (max-width: 1050px) {
  .margined {
    /*background-color: lightblue;*/
    width: 98%;
    margin-left: auto;
    margin-right: auto;
  }
}
```

**Comment by towerofnix** [comment]
yes, i think that would work pretty well! (tested via inspect element and it looks good!)

**Comment by jeffalo** [comment]
> yes, i think that would work pretty well! (tested via inspect element and it looks good!)

awesome. the only problem is when i asked people on the ocular thread, they really didn't how much space would be wasted. i wonder if i pushed this if anyone would really care that much though.

**Comment by towerofnix** [comment]
i saw some comments there and their concerns are valid—i think its probably not too bad as is currently, and userstyles can go enough of the way to adjust things to my own preferences :P

to be honest i think boosting the font size to a bigger default size (or simply not changing it from the browser default, usually 16px but possibly larger or smaller) would be just as useful, but that would probably also be worth checking in with other users about :package:

**Comment by jeffalo** [comment]
thanks. i don't remember why i changed the font size, i'll see if keeping it as browser default messes with anything.

**Comment by mxmou** [comment]
I actually thing using the same margins as Scratch would be a good idea if the layout of posts was exactly the same as Scratch's (so the left part with the profile picture would have a with of `208px` instead of `20%`).
